packages/graphql-codegen-cli/src/load.ts

```diff
@@ -28,7 +28,7 @@ export const defaultDocumentsLoadOptions = {
 };
 
 export async function loadSchema(
-  schemaPointers: UnnormalizedTypeDefPointer,
+  schemaPointers: UnnormalizedTypeDefPointer | UnnormalizedTypeDefPointer[],
```

Comment:
Should we add tests for this use case?

Reply:
Good idea! This is done in https://github.com/dotansimha/graphql-code-generator/pull/10227/commits/9efc60b7e075fce6ef74b79d4bb27803ce4115ec

Note that instead of testing this `load.ts` directly, I'm passing multiple schemas into Codegen Context which eventually passes those schemas to `load.ts`